spec/user_spec.rb

```diff
@@ -127,50 +126,53 @@
   end 
 
   context '#cleanup_old_experiments!' do
-    let(:user_keys) do
```

Comment:
Ln 130 - 175 changes: group the tests in a describe `when the user is in the experiment without version number`. Added extra key: `'link_color:external_key' => "value",`